build.gradle

```diff
@@ -165,6 +165,7 @@ def validationXsdPath = project(':sbe-tool').projectDir.toString() + '/src/main/
 project(':sbe-tool') {
     dependencies {
         compile 'org.agrona:agrona:0.9.6'
+        compile 'com.google.guava:guava:21.0'
```

Comment:
Guava's use here is to transform names into the appropriate casing for Rust constructs. Most notably, lower_snake_case.  That sort of string transformation seemed to not be a core competency or goal of the SBE tool, so I thought it appropriate to bring in a reputable 3rd party open source library backed by Google, managed by the pre-existing dependency system in gradle.

If you prefer, I can replicate the case-transformation code here in this repo, at the cost of increased local code weight (read: maintenance and testing costs).